example/postgres/distros/DEMO.md

```
@@ -0,0 +1,77 @@
+# PostgreSQL-distro SBoB demo — deploy, bind, contrast
+
+Bring up the fork stack from the bob repo root first:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)